person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  att_reader :id
+
+  attr_accessor :name, :age
```

Comment:
- [ ] Optional: It is always a best practice to place the getter & setter before initialize method.

Reply:
Alright Thanks very much the review @sja-thedude